media-library/overview/rclone.md

```diff
@@ -0,0 +1,138 @@
+# Rclone
+
+![](../../.gitbook/assets/rclone/rclone-logo.svg)
+
+[Rclone](https://rclone.org/) is a command-line program that helps manage files on a cloud storage. It supports [over 70 cloud storage products](https://rclone.org/#providers) and has powerful cloud equivalents to the unix commands rsync, cp, mv, mount, ls, ncdu, tree, rm, and cat.
```

Comment:
Start by first talking about why someone might need it. This guide is basically for technology team who wants to migrate large number of assets to ImageKit DAM. For small number of assets our existing upload from many providers through uppy (onedrive/Google Drive/box etc) is enough.